_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Bitcoin Optech Newsletter #98'
+permalink: /en/newsletters/2020/05/20/
+name: 2020-05-20-newsletter
+slug: 2020-05-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter relays a request for comments on a proposed
+change to the BIP341 taproot transaction digest and briefly summarizes
+discussion about a new and more concise protocol for atomic swaps.  Also
+included are our regular sections describing changes to services and
+client software, new releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+- **Evaluate proposed changes to BIP341 taproot transaction digest:** as
+  described in [last week's newsletter][news97 spk commit], there has
+  been a request for [taproot][topic taproot] signatures to make an
+  additional commitment to all the scriptPubKeys of the UTXOs being
```

Comment:
nit: s/all the scriptPubKeys of the UTXOs/the scriptPubKeys of all the UTXOs/ sounds slightly better to me (the first sounds to me a bit like a UTXO could include multiple scriptPubKeys).